cpu/ebizzy.py

```diff
@@ -68,6 +68,12 @@ def setUp(self):
     # Note: default we use always mmap()
     def test(self):
 
+        perfstat = self.params.get('perfstat', default='')
+        if len(perfstat):
+                perfstat = 'perf stat ' + perfstat
+        taskset = self.params.get('taskset', default='')
+        if len(taskset):
```

Comment:
you can use `if taskset:` here

Reply:
Thanks for Review, Harish. Sorry for the delayed response.
I have addressed all the suggestions made by you. And the checks have passed which had not passed earlier due to indentation(as you had pointed out earlier). Please merge this if nothing more is needed.